src/js/state/ConfigPropValues/test.ts

```diff
@@ -0,0 +1,110 @@
+import initTestStore from "src/js/test/initTestStore"
+import ConfigPropValues from "./index"
+
+const configName1 = "testconfig1"
+const configName2 = "testconfig2"
+const configProperty1 = "testproperty1"
+const configProperty2 = "testproperty2"
+
+const data1 = {
+  test: "test"
+}
+
+const data2 = "test2"
+
+let select, dispatch, store
+beforeEach(() => {
+  store = initTestStore()
+  dispatch = store.dispatch
+  select = (f) => f(store.getState())
+})
+
+test("Create or Update (Upsert)", () => {
+  // add one prop to one config
+  dispatch(
+    ConfigPropValues.set({
+      configName: configName1,
+      propName: configProperty1,
+      value: data1
+    })
```

Comment:
What do you think about making the config values a flat object, and we trust that the keys will be prefixed properly? I think it would make for a more clear, less verbose api if it looked like: `ConfigValues.set(key, value)`